.github/workflows/ci.yml

```diff
@@ -5,10 +5,15 @@ jobs:
   job1:
     name: trunner.nim
     runs-on: ubuntu-latest
-    container: nimlang/nim:latest
 
     steps:
     - uses: actions/checkout@v2
 
-    - name: Compile and run `tests/trunner.nim`
-      run: "nim c -r --styleCheck:error --hint[Processing]:off tests/trunner.nim"
+    - name: Download all nim exercises
+      run: git clone --depth 1 https://github.com/exercism/nim.git
+
+    - name: Build image
+      run: docker build -t nimtestrunner .
+
+    - name: Run image to test all exercises mounted from github repo
+      run: docker run -v ${PWD}/nim/:/mnt/nim nimtestrunner
```

Comment:
I love that you test the test runner by running it on the existing exercises!

Reply:
Thanks

Reply:
Yeah, it's a good sanity test. Thanks to @ynfle for getting it working in the container - the image doesn't contain `git`, of course.

I originally wrote that test just to tide us over until we add some more comprehensive tests. But I can't name the concrete advantage of testing every Nim exercise vs just testing only the `hello-world` exercise and a file that contains `import re` (to check that PCRE is available in the image).

If you love it then we'll keep running it. There's a minor cost that it adds 2 minutes to CI, but we can use two jobs and keep one fast.